cli/cmd/plugin/ui/web/tanzu-ui/src/App.tsx

```diff
@@ -1,16 +1,16 @@
 // React imports
-import React, {useContext, useEffect} from 'react';
+import React, { useContext, useEffect } from 'react';
 import { Routes, Route } from 'react-router-dom';
 
 // App imports
-import HeaderComponent from './shared/components/header/header.component';
-import SideNavigationComponent from './shared/components/side-navigation/SideNavigation.component';
-import GetStartedComponent from './views/getting-started/getting-started.component';
+import HeaderBarComponent from './shared/components/header/HeaderBar';
```

Comment:
should we use HeaderBar instead of HeaderBarComponent?

Reply:
yes, thanks for catching this